src/posix.mak

```diff
@@ -494,8 +497,11 @@ install: all
 
 ######################################################
 
-checkwhitespace: $(HOST_DMD_PATH)
-	CC=$(HOST_CXX) $(HOST_DMD_RUN) -run checkwhitespace.d $(SRC) $(GLUE_SRC) $(ROOT_SRCS)
+checkwhitespace: $(HOST_DMD_PATH) $(TOOLS_DIR)
```

Comment:
(N.B. We also have a makefile in tools, but we don't want to make the build of dmd depend on building all tools for obvious reasons.)